static/js/containers/ChannelPage.js

```diff
@@ -36,21 +49,30 @@ class ChannelPage extends React.Component {
   }
 
   renderContents(channelName, channels, postsForChannel, posts) {
+    if (!channels.data || !posts.data) {
+      return
+    }
     const channel = channels.data.get(channelName)
+    if (!postsForChannel.data) {
+      return
+    }
     const postIds = postsForChannel.data.get(channelName)
 
-    if (R.isNil(channel) || R.isNil(postIds)) {
+    if (!channel || !postIds || !posts.data) {
```

Comment:
Seems like we could get some use out of a utility function that returns true if any nil args are passed in:

```javascript
function anyNull() {
  return R.any(R.isNil, arguments)
}
```

You can take or leave that. Not too important